src/lib/Scenes/Artwork/Components/ArtworkActions.tsx

```diff
@@ -32,11 +32,35 @@ export class ArtworkActions extends React.Component<ArtworkActionsProps> {
     })
   }
 
+  handleArtworkShare = async () => {
+    const { title, id, artists } = this.props.artwork
+    let message
+    if (artists && artists.length) {
+      const names = []
+      artists.forEach((artist, i) => {
+        if (i < 3) {
+          names.push(artist.name)
+        }
+      })
+      message = `${title} by ${names.join(", ")} on Artsy http://artsy.net/artwork/${id}`
```

Comment:
#minor but these should probably all be `https`.

Also, it seemed non-trivial to add tests for the share functionality but would be good to have some unit tests for the logic that forms this message!

Reply:
Related-- this `id` will be the global ID so we should either query for `href` or use the `gravityID` here.